DFE-Digital/teacher-services-cloud
# [195] Split cluster creation and configuration

## What
Issue when recreating cluster in https://github.com/DFE-Digital/teacher-services-cloud/actions/runs/4233797190/jobs/7355225046

Related to: https://itnext.io/terraform-dont-use-kubernetes-provider-with-your-cluster-resource-d8ec5319d14a

## How to review
- Deploy dev cluster
- Change default pool vm type to Standard_D2_v3
- Apply: it should fail
- Follow process in Readme
